db/db.js

```diff
@@ -294,6 +294,52 @@ async function updateSessionStillCounter(stillcounter, sessionid, locationid) {
   }
 }
 
+async function isOverdoseSuspectedInnosent(session, location) {
+  const now = new Date()
+  const start_time_sesh = new Date(session.start_time)
+
+  // Current Session duration so far:
+  const sessionDuration = (now - start_time_sesh) / 1000
+
+  // threshold values for the various overdose conditions
+  const still_threshold = location.still_threshold
+  const sessionDuration_threshold = location.duration_threshold
+
+  // number in front represents the weighting
+  const condition2 = 1 * (session.still_counter > still_threshold) // seconds
+  const condition3 = 1 * (sessionDuration > sessionDuration_threshold)
+
+  let alertReason = 'NotSet'
```

Comment:
Can you please make these alertReasons into an Enum and then use them everywhere they are hardcoded?